src/transformers/models/vit/feature_extraction_vit.py

```diff
@@ -0,0 +1,136 @@
+# coding=utf-8
+# Copyright Google AI and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Feature extractor class for ViT."""
+
+from typing import List, Union
+
+import numpy as np
+import torch
+from PIL import Image
+from torchvision import transforms as T
```

Comment:
I'm not super fan of the one-letter import and we try to avoid it when possible. Here we could just import the things we need from `torchvision.transforms`: `ToPILImage`, `Resize`, `ToTensor`, `Normalize` and `Compose`.